modules/ROOT/pages/installation/linux/rpm.adoc

```diff
@@ -31,33 +31,42 @@ Installation instructions can be found on the manufacturer's website:
 [[linux-rpm-install-standard]]
 === Set up the repository
 
-To add the Neo4j repository to the package manager, run the following command as a sudo user:
-
+. Import the Neo4j GPG public key into the system’s RPM keyring.
+The key is required to verify the authenticity of the Neo4j packages you will install.
++
 [source, shell, subs="attributes"]
 ----
 rpm --import https://debian.neo4j.com/neotechnology.gpg.key
 ----
 
-Create a `neo4j.repo` file:
-
+. Create a `neo4j.repo` file in the `/etc/yum.repos.d/` directory.
+This file contains the repository configuration for Neo4j.
++
 [source, shell, subs="attributes"]
 ----
-cat << EOF >  /etc/yum.repos.d/neo4j.repo
+cat <<EOF > /etc/yum.repos.d/neo4j.repo
```

Comment:
for some reason the `<EOF >` is missing in the rendered page, which makes this code wrong:
https://neo4j-docs-operations-2475.surge.sh/operations-manual/2025.07/installation/linux/rpm/

I have no idea why your change would make that happen though :(